chemprop/models/ffn.py

```diff
@@ -0,0 +1,355 @@
+from typing import List, Tuple
+
+import torch
+import torch.nn as nn
+import re
+
+
+class MultiReadout(nn.Module):
+    """A fake list of FFNs for reading out as suggested in
+    https://discuss.pytorch.org/t/list-of-nn-module-in-a-nn-module/219/3"""
```

Comment:
this link is broken

Reply:
This should be removed when I replace `AttrProxy()` with `nn.ModuleList()`. I will remove it.